example/android/settings.gradle

```diff
@@ -1,11 +1,4 @@
 rootProject.name = 'BitmovinPlayerReactNativeExample'
 apply from: file("../node_modules/@react-native-community/cli-platform-android/native_modules.gradle"); applyNativeModulesSettingsGradle(settings)
 include ':app'
-
-include ':bitmovin-player-react-native'
```

Comment:
We don't need this anymore?

Reply:
Nope. This now auto-magically happens with the auto-linking support. Finding out about this was the biggest paint point during this migration, since [this](https://github.com/bitmovin/bitmovin-player-react-native/pull/318#discussion_r1395444381) is needed for gradle auto linking to work. 

You can read a bit about how this works here: https://github.com/react-native-community/cli/blob/main/docs/autolinking.md#platform-android